code/__DEFINES/mobs.dm

```diff
@@ -386,6 +386,8 @@
 #define PULL_PRONE_SLOWDOWN 1.5
 #define HUMAN_CARRY_SLOWDOWN 0.35
 
+#define DO_AFTER_SLOWDOWN 2
```

Comment:
is this during any doafter?

Reply:
leftover from when I was handling slowdown inside the do_after, I'll get rid of it